<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PomodoroManualAddDialog</class>
 <widget class="QDialog" name="PomodoroManualAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Pomodoro</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBoxConfirmOrCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPickTodoItem">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pick associated Todo Item</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditPomodoroDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.mm.yyyy</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditPomodoroStartTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>hh:mm</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditPomodoroFinishTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>hh:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPickDate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Only adding Pomodoros for existing Todo items is now supported</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxConfirmOrCancel</sender>
   <signal>accepted()</signal>
   <receiver>PomodoroManualAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxConfirmOrCancel</sender>
   <signal>rejected()</signal>
   <receiver>PomodoroManualAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditPomodoroDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>timeEditPomodoroStartTime</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditPomodoroDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>timeEditPomodoroFinishTime</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
